docs/synthetics/http-monitor.md

```diff
@@ -1,24 +1,24 @@
-title: HTTP Monitor
-description: HTTP monitor can monitor any HTTP endpoint like API, Web URL, etc. 
+title: HTTP Monitor  
+description: HTTP monitor can monitor any HTTP endpoint like API, Web URL, etc.
 
 The HTTP monitor sends a single HTTP request to the specified URL with configured request settings. It records the response data like headers, body, error if any and the various timings.
 
 ## Configuration
 
 ### General
 
-* **Name** - Name of the monitor. Max length is 255 characters.
```

Comment:
Looks like some characters were added/inserted here and in several places below.Not sure if accidentally or on purpose or if that changes how things are rendered or not.  JFYI.

Reply:
Yes I don't know how this happened, I will remove them